lib/sigsci-agent.sh

```diff
@@ -87,35 +87,43 @@ then
 
   ## install agent
 
-  # if agent version not specified then get the latest version.
-  if [ -z $SIGSCI_AGENT_VERSION ]
+  #  if we're downloading from an internal repo, then we don't need to worry about figuring out what version we need
```

Comment:
`# if a specific download url is provided, we can bypass version-determination logic`

might be a little clearer